fixtures/chauvet-professional/ovation-p-56-f.json

```diff
@@ -0,0 +1,109 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Ovation P-56 F",
+  "categories": ["Other"],
+  "meta": {
+    "authors": ["van egmond"],
+    "createDate": "2024-05-13",
+    "lastModifyDate": "2024-05-13"
+  },
+  "links": {
+    "manual": [
+      "https://www.chauvetprofessional.com/wp-content/uploads/2015/01/Ovation_P-56FC_UM_Rev9.pdf"
+    ],
+    "productPage": [
+      "https://www.chauvetprofessional.com/products/ovation-p-56fc/"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?time_continue=10&v=yN8PjMBAPBo&embeds_referring_euri=https%3A%2F%2Fwww.chauvetprofessional.com%2F&embeds_referring_origin=https%3A%2F%2Fwww.chauvetprofessional.com&source_ve_path=Mjg2NjY&feature=emb_logo"
+    ]
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "fineChannelAliases": ["Dimmer fine"],
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Red": {
+      "fineChannelAliases": ["Red fine"],
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "fineChannelAliases": ["Green fine"],
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "fineChannelAliases": ["Blue fine"],
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "Amber": {
+      "fineChannelAliases": ["Amber fine"],
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Amber"
+      }
+    },
+    "Lime": {
+      "fineChannelAliases": ["Lime fine"],
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Lime"
+      }
+    },
+    "Strobe": {
+      "capability": {
+        "type": "NoFunction"
+      }
+    },
+    "Virtual color wheel": {
+      "capability": {
+        "type": "ColorPreset"
+      }
```

Comment:
```suggestion
      "capabilities": [
        {
          "dmxRange": [0, 7],
          "type": "NoFunction"
        },
        {
          "dmxRange": [8, 15],
          "type": "ColorPreset",
          "comment": "C3050-Md Yellow"
        },
        {
          "dmxRange": [16, 23],
          "type": "ColorPreset",
          "comment": "C3040-Lt Yellow"
        },
        {
          "dmxRange": [24, 31],
          "type": "ColorPreset",
          "comment": "C3240-Amb Yellow"
        },
        {
          "dmxRange": [32, 39],
          "type": "ColorPreset",
          "comment": "C2340-VLt Amber"
        },
        {
          "dmxRange": [40, 47],
          "type": "ColorPreset",
          "comment": "C2040-Lt Amber"
        },
        {
          "dmxRange": [48, 55],
          "type": "ColorPreset",
          "comment": "C2050-Md Amber"
        },
        {
          "dmxRange": [56, 63],
          "type": "ColorPreset",
          "comment": "C2060-Dk Amber"
        },
        {
          "dmxRange": [64, 71],
          "type": "ColorPreset",
          "comment": "C1050-Lt Red"
        },
        {
          "dmxRange": [72, 79],
          "type": "ColorPreset",
          "comment": "C1080-Md Red"
        },
        {
          "dmxRange": [80, 87],
          "type": "ColorPreset",
          "comment": "C1020-NC Pink"
        },
        {
          "dmxRange": [88, 95],
          "type": "ColorPreset",
          "comment": "C1030-Md Pink"
        },
        {
          "dmxRange": [96, 103],
          "type": "ColorPreset",
          "comment": "C1630-Dk Pink"
        },
        {
          "dmxRange": [104, 111],
          "type": "ColorPreset",
          "comment": "C1250-Md Red Amber"
        },
        {
          "dmxRange": [112, 119],
          "type": "ColorPreset",
          "comment": "C1060-Dk Red Amber"
        },
        {
          "dmxRange": [120, 127],
          "type": "ColorPreset",
          "comment": "C1650-Magenta"
        },
        {
          "dmxRange": [128, 135],
          "type": "ColorPreset",
          "comment": "C6170-Dk Magenta"
        },
        {
          "dmxRange": [136, 143],
          "type": "ColorPreset",
          "comment": "C6020-Lt Lavender"
        },
        {
          "dmxRange": [144, 151],
          "type": "ColorPreset",
          "comment": "C5030-Lt Blue"
        },
        {
          "dmxRange": [152, 159],
          "type": "ColorPreset",
          "comment": "C5020-VLt Blue"
        },
        {
          "dmxRange": [160, 167],
          "type": "ColorPreset",
          "comment": "C5430-Lt Blue2"
        },
        {
          "dmxRange": [168, 175],
          "type": "ColorPreset",
          "comment": "C5070-Blue"
        },
        {
          "dmxRange": [176, 183],
          "type": "ColorPreset",
          "comment": "C5050-Md Blue"
        },
        {
          "dmxRange": [184, 191],
          "type": "ColorPreset",
          "comment": "C5060-Dk Blue"
        },
        {
          "dmxRange": [192, 199],
          "type": "ColorPreset",
          "comment": "C5690-Indigo"
        },
        {
          "dmxRange": [200, 207],
          "type": "ColorPreset",
          "comment": "C5080-VDk Blue"
        },
        {
          "dmxRange": [208, 215],
          "type": "ColorPreset",
          "comment": "C5081-VdK Blue2"
        },
        {
          "dmxRange": [216, 215],
          "type": "ColorPreset",
          "comment": "C4370-Yel Green"
        },
        {
          "dmxRange": [224, 231],
          "type": "ColorPreset",
          "comment": "C4070-Green"
        },
        {
          "dmxRange": [232, 239],
          "type": "ColorPreset",
          "comment": "C4550-Turquoise"
        },
        {
          "dmxRange": [240, 247],
          "type": "ColorPreset",
          "comment": "C4560-Aqua"
        },
        {
          "dmxRange": [248, 255],
          "type": "ColorPreset",
          "comment": "C4570-Blue Green"
        }
      ]
```

It's not officially documented which color is at which value, but there's 31 colors so I assume they're evenly spaced and the first slot is open.

The manual has RGBAL values for each of these, but there isn't any way yet (see #437) to encode RGBAL in OFL.